index.js

```diff
@@ -1,5 +1,16 @@
 'use strict';
 
+exports.getQueryString = function (maybeUrl) {
+	if (!maybeUrl) {
+		return;
+	}
+
+	var pos = maybeUrl.indexOf('?');
+	if (pos !== -1) {
+		return maybeUrl.substring(pos + 1);
+	}
+};
```

Comment:
I think that's actually beneficial. It should return a string as long as the input is a string. And it should throw if the input is not a string, so:

```js
exports.extract = function (str) {
    return str.split('?')[1] || '';
};
```